kubernetes/client/ws_client.py

```diff
@@ -191,6 +191,14 @@ def run_forever(self, timeout=None):
             while self.is_open():
                 self.update(timeout=None)
 
+    def close(self, **kwargs):
```

Comment:
Also, just out of curiosity, should we return True/False if we were able to close successfully the socket?